### DISCUSSION TITLE
LSPosed repo do not publish release

### DISCUSSION THREAD

**Discussion by MichaelZhuravsky** [LSPosed/LSPosed#1347] [Q&A] [answered]
Hello!
I have a [repo](https://github.com/Xposed-Modules-Repo/ru.mike.mcroot) and have made the first release. But up to now I cannot find the module via [repo](https://modules.lsposed.org)

So, what should I do to find what's wrong and what I need to do to fix it?

**Comment by JasonKhew96** [comment]
> Hello! I have a [repo](https://github.com/Xposed-Modules-Repo/ru.mike.mcroot) and have made the first release. But up to now I cannot find the module via [repo](https://modules.lsposed.org)
> 
> So, what should I do to find what's wrong and what I need to do to fix it?

Support/Discussion URL missing
![image](https://user-images.githubusercontent.com/29531167/139685487-2d589b29-0a5e-48c8-bdad-039e2a2f2b00.png)

**Reply by MichaelZhuravsky** [reply]
Thx! But I have two URLs. I need to choose the one only?

**Comment by yujincheng08** [comment, marked as answer]
It has been added. It maybe because of cache that you cannot see it yet: https://github.com/Xposed-Modules-Repo/modules/runs/4068437544?check_suite_focus=true#step:7:39

**Comment by MichaelZhuravsky** [comment]
I've seen the module via LSPosed Manager repo's page, but could not find via the [web](https://modules.lsposed.org).
Thx anyway
P.S. There is full-text search only. My fault :-(

**Reply by yujincheng08** [reply]
https://modules.lsposed.org/module/ru.mike.mcroot
